CHANGELOG.md

```diff
@@ -117,6 +117,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### 🛠 Maintenance
 
  - Update oui to 1.12 ([#7865](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7865))
+ - Bump OpenSearch-Dashboards to use NodeJS v18.20.4 ([#8294](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8294))
```

Comment:
This bump is CVE-related so it should be under `security`

We should also change the language and structure to include the CVEs or at least the HIGH ones:
``` 
- [CVE-2024-27980][CVE-2024-27983] Use Node.js v18.20.4 ([#8294](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8294))
```

Reply:
Done in latest commit :D